server/models/requestGroupModel.ts

```diff
@@ -6,7 +6,7 @@ interface RequestGroupInterface {
   description: string
```

Comment:
I realize this isn't your change but shouldn't the deleted field be here as well?

Reply:
Yes! good catch :smiley_cat:

Reply:
Also in the schema.ts too. I'll need to make these changes for requestType as well.

Reply:
^ yup! Added in schema and `RequestGroupsDataSource`